domains/baldeagle.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "SniverDaBest",
+        "discord": "919385800364425256"
+    },
+    "record": {
+        "URL": "www.youtube.com/watch?v=dQw4w9WgXcQ"
```

Very funny link (Won't merge until you change the link to something else)
```suggestion
        "URL": "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
```